grafana/provisioning/dashboards/cadvisor.json

```diff
@@ -0,0 +1,814 @@
+{
```

Comment:
What is this file? Some sort of dashboard template?

Reply:
It is used for benchmark tests. @hadjiszs could say more

Reply:
(answering in the other comment [here](https://github.com/topos-protocol/topos/pull/397#discussion_r1422752706))